### ISSUE TITLE
Upgrading Legendaries with LiteBag open causes an error.

### ISSUE THREAD

**Issue by arcadepro** [xod-wow/LiteBag#5]
Open bag. Talk to Runecarver. Select upgrade legendary item. Pick any legendary from the central button flyout. Error is triggered.

```[ADDON_ACTION_FORBIDDEN] AddOn 'LiteBag' tried to call the protected function 'SpellStopCasting()'.
[string "@Interface\AddOns\AdiButtonAuras\libs\BugGrabber\BugGrabber.lua"]:519: in function <...AddOns\AdiButtonAuras\libs\BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `SpellStopCasting'
[string "@Interface\AddOns\Blizzard_RuneforgeUI\Blizzard_RuneforgeCraftingFrame.lua"]:187: in function <...zard_RuneforgeUI\Blizzard_RuneforgeCraftingFrame.lua:168>
[string "=(tail call)"]: ?
[string "@Interface\SharedXML\CallbackRegistry.lua"]:55: in function `TriggerEvent'
[string "@Interface\AddOns\Blizzard_RuneforgeUI\Blizzard_RuneforgeItemSlot.lua"]:225: in function `SetItem'
[string "@Interface\AddOns\Blizzard_RuneforgeUI\Blizzard_RuneforgeCraftingFrame.lua"]:127: in function `SetItem'
[string "@Interface\AddOns\Blizzard_RuneforgeUI\Blizzard_RuneforgeCraftingFrame.lua"]:21: in function `onClickFunc'
[string "@Interface\FrameXML\EquipmentFlyout.lua"]:543: in function <Interface\FrameXML\EquipmentFlyout.lua:540>
```

I don't think the above log is helpful but the steps to reproduce are very simple. I think it's probably something to do with the items getting highlighted in your bag during this process.

**Comment by mbattersby** [comment]
Hmm. I don't know what to make of this and I don't have any easy way to test it. Hopefully I will remember to try it out if and when I next upgrade a Legendary.

It seems incredible to me that I would not have had my bags open when I upgraded one, but perhaps I did not.

**Comment by mbattersby** [comment]
There is a small chance that this is actually dropdown menu taint. LiteBag 9.0.12 has an attempted fix for some of the taint in it, so that might confuse debugging.

**Comment by mbattersby** [comment]
How the hell is SpellStopCasting() having any interaction with LiteBag?

**Comment by mbattersby** [comment]
The !TaintFIx stuff is live in litebag now. I never had this happen and still have no way to test it.

**Comment by mbattersby** [comment]
99.9% sure this is the same issue as #5 and is now fixed.

**Comment by mbattersby** [comment]
Fixed in 9.2.1 by using LibUIDropDownMenu. Pending some testing if I have a legendary to upgrade.

**Comment by mbattersby** [comment]
Seems to be ok, closing.
